frontend/app/src/components/defi/wizard/DefiWizard.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <v-card>
+    <v-stepper v-model="step" vertical>
+      <v-stepper-step :complete="step > 1" step="1">
+        Setup DeFi
+      </v-stepper-step>
+      <v-stepper-content step="1">
+        <v-card class="mb-12" height="200px" outlined color="grey lighten-4">
+          <v-card-title>
+            Select options to improve your Rotki DeFi experience
```

Comment:
I know I am also guilty of not being consistent but we have been trying to keep it consistently lowercase. "rotki". Right?